src/Zope2/Startup/serve.py

```diff
@@ -18,6 +18,7 @@
 import os
 import re
 import sys
+import Zope2
```

Comment:
This import needs to go below `from App.config import getConfiguration` to make the linting tests pass. I would have done it myself but since you are working off a fork I can't.

Reply:
Sorry. Done. 

But the PR has checked the option "allow edit from mainteners" (that's the default) where I read "If checked, users with write access to zopefoundation/Zope can add new commits to your master branch."
Anyway, if is a better choice, next time I'll do a branch instead of working on a fork in my account. No problem.

Reply:
In didn't even know there is such an option. I never work with forks, so my automatic reaction is "this sits in someone elses's account, I have no access". But even with that option, I would be forced to do a separate clone and then build to get your fork. That's really not necessary. If possible, create a branch in here. You will make everyone's life easier.